pkg/stub/handler.go

```diff
@@ -438,7 +451,7 @@ func (h *Handler) manageDockerCfgSecret(deleted bool, samplesResource *v1alpha1.
 	}
 
 	h.registrySecret = secret
-	h.samplesResource = samplesResource
```

Comment:
why is this  no longer needed?  or was it already dead code?

Reply:
it was not needed previously ... and in general I want to minimize where we set `h.samplesResource`